modules/gltf/register_types.cpp

```diff
@@ -48,6 +46,8 @@
 #include "gltf_spec_gloss.h"
 #include "gltf_state.h"
 #include "gltf_texture.h"
+#include "modules/gltf/editor_scene_exporter_gltf_plugin.h"
+#include "modules/gltf/editor_scene_importer_gltf.h"
```

Comment:
In this case it's wrong, modules are made to be modular and should not hardcode their absolute path. It should still compile without change if the module was renamed or moved out of tree.